test/e2e/scenario/recorder/recorder.scenario.ts

```diff
@@ -7,7 +7,8 @@ import type {
 } from '@datadog/browser-rum/src/types'
 import { NodeType, IncrementalSource, MouseInteractionType } from '@datadog/browser-rum/src/types'
 
-import { FrustrationType } from '@datadog/browser-rum-core'
+// Import from src to have properties of const enums
+import { FrustrationType } from '@datadog/browser-rum-core/src/rawRumEvent.types'
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why are we missing enums properties from the package import?

Reply:
Since `FrustrationType` is exported from `@datadog/browser-rum-core`, what about using:

```ts
export const FrustrationType = {
  RAGE_CLICK: 'rage_click',
  ...
}
```